### PR TITLE
New Function , updatet Readme 

### PR DESCRIPTION
**Pull request by H4xl0r**
Portals are able now to send mails via the myportal.php
via calling 

`this->sendmail($sub, $bod, $sender, $email);`
where

 ```
$sub, $bod, $sender, $email is:
 subject ,body , sendermail ,recivermail.

```

### CODE REVIEW

**Comment by H4xl0r** [comment]
@frozenjava PULL PULL PULL

**Comment by frozenjava** [comment]
Thanks for the PR. I'll test it this week and merge in if no changes are required.

**Comment by H4xl0r** [comment]
@frozenjava u can use this portal for testing : 
https://github.com/H4xl0r/evilportals/tree/master/portals/mailtoken

But setup smtp settings first (readme update )

**Comment by H4xl0r** [comment]
still no reaction :-(

**Comment by frozenjava** [comment]
I had some time to review this evening and I have a couple points of feed back:

* Technically the next version will be 3.3 not 3.1
* Although I think the portals repo maintained by you and kbeflo is cool, I don't want this project to link to them because they do host phishing content and I don't want this repo to in anyway encourage or link to phishing.

So if you can remove those things from the README that would be great and I'll review again!

File: README.md [hunk 1]
```diff
@@ -33,22 +33,73 @@ scp -r EvilPortal root@172.16.42.1:/pineapple/modules/
 
 Head on over to the Wifi Pineapples Web Interface and go to the Evil Portal module. You're all done!
 
+## Portal Sendmail function needs to be setup ! (optional)
```
Comment (frozenjava) [inline README.md:36]: Maybe add this to the wiki instead of the readme?

File: README.md [hunk 2]
```diff
@@ -33,22 +33,73 @@ scp -r EvilPortal root@172.16.42.1:/pineapple/modules/
...
+* writelog function refiew (looks like its not working ? )
+* redirect function refiew (for calling without $target eg refresh page) 
+* documenatation for portals and functions / calls 
 
 ## Release History
+* Figured out how to redirect clients going to HTTPS sites 
+* Added Mail function via sendmail , Portals can send emails now !
 
-### Version 3.1
+### Version 3.1 (beta)
```
Comment (frozenjava) [inline README.md:102]: version 3.1 and 3.2 have already been release, next version is 3.3.

File: README.md [hunk 3]
```diff
@@ -33,22 +33,73 @@ scp -r EvilPortal root@172.16.42.1:/pineapple/modules/
...
+AuthPass=your_gmail_password
+
+# Use SSL/TLS certificate to authenticate against smtp host.
+#UseTLSCert=YES
+
+# Use this RSA certificate.
+#TLSCert=/etc/ssl/certs/ssmtp.pem﻿
+```
+
+Example for Portal with sendmail :
```
Comment (frozenjava) [inline README.md:75]: These are cool projects, but I don't want them referenced from here because they host phishing content.

**Review by frozenjava — CHANGES_REQUESTED** [review]
Technically the next version will be 3.3 not 3.1
    Although I think the portals repo maintained by you and kbeflo is cool, I don't want this project to link to them because they do host phishing content and I don't want this repo to in anyway encourage or link to phishing.

**Comment by H4xl0r** [comment]
Will repull

**Pull request closed without merging**